src/mococrw/hsm.h

```diff
@@ -53,6 +55,36 @@ class HSM
      * @param keyID The ID of the private key to load.
      */
     virtual openssl::SSL_EVP_PKEY_Ptr loadPrivateKey(const std::string &keyID) = 0;
+
+    virtual openssl::SSL_EVP_PKEY_Ptr genKeyGetPublic(const RSASpec &spec,
```

Comment:
This should have doxygen user documentation here. What do the parameters mean? What kind of input is allowed? How does this translate into PKCS#11 as we have PKCS#11 in mind. What would we do with it otherwise? How do we explain that the load functions just support a key id? Should it maybe support more? At least the label?

Reply:
Done. Still wondering if error reporting makes sense. Lots of things happen in libp11 and there's not clean way to pinpoint what went from MoCOCrW.